.github/workflows/fabric_acctest.yml

```diff
@@ -0,0 +1,155 @@
+name: Acceptance Tests
+# This workflow determines whether a PR comes from an external fork
+# (which requires approval from us) or from a branch on this repository
+# (which means it was made by us and can run immediately).  Once a PR
+# is approved, the PR code gains access to secrets referenced in this
+# workflow.
+
+# The 'build' job and subsequent jobs, are executed only when the pull
+# request is not a draft, regardless of whether it is from an internal
+# branch or external fork.
+
+# Any changes to this job, even from internal contributors, require heavy scrutiny.
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'LICENSE'
```

Comment:
If we stick with having separate acceptance test workflows for each service, then it would probably be better to use a `paths:` option and specify the Fabric-relevant path patterns, so you're not running acceptance tests for PRs that only touch NE resources.

Reply:
Agree. Implementing.